src/apps/chifra/cmd/receipts.go

```diff
@@ -24,7 +27,7 @@ var receiptsCmd = &cobra.Command{
 	Short:   shortReceipts,
 	Long:    longReceipts,
 	Version: versionText,
-	RunE:    receiptsPkg.RunReceipts,
+	RunE:    runWithFileSupport,
```

Comment:
You may have though of this already, but perhaps `runWithFileSupport` could take a function pointer to `receiptsPkg.RunReceipts`. In that way, all commands could use the same `runWithFileSupport`.

Reply:
Yes, this is exactly what I want to do 👍